index.js

```diff
@@ -56,16 +56,31 @@ class NpmAddDependencies {
 
   runNpmShow(dep) {
     return new Promise((resolve) => {
-      npmRun.exec(`npm show ${dep} dist-tags`, (err, stdout) => {
+      npmRun.exec(`npm show ${dep} dist dist-tags`, (err, stdout) => {
```

Comment:
Why do you need the `dist` flag here?

Reply:
I thougt need the `dist` flag for exact version match, we could get only latest version with `dist-tags` flag.

```bash
$ npm show lodash@4.0.0 dist-tags
{ latest: '4.17.15' }

$ npm show lodash@4.0.0 dist dist-tags
dist =
{
  shasum: '9ac43844c595e28d30108b7ba583703395922dfc',
  tarball: 'https://registry.npmjs.org/lodash/-/lodash-4.0.0.tgz'
}
dist-tags = { latest: '4.17.15' }
```

But nothing will be displayed if specified version is not exist. I think over don't need `dist` flag.

```bash
$ npm show lodash@4.17.99 dist dist-tags

$
```